fbinstant/manifests/web/engine_template.html

```diff
@@ -12,19 +12,18 @@
     </style>
 </head>
 <body oncontextmenu="return false;">
-    <script src="https://connect.facebook.net/en_US/fbinstant.6.3.js"></script>
+    <script src="https://connect.facebook.net/en_US/fbinstant.6.3.js"/>
     <script type='text/javascript'>
-        // Set up a progress listener and feed progress to FBInstant
-        Progress.updateProgress = function (percentage, text) {
-            FBInstant.setLoadingProgress(percentage);
-        }
-
         // Do early initialization of FBInstant
         // This is required to be able to properly update the loading
         // progress above.
         FBInstant.initializeAsync().then(function() {
             // This will be checked by the FBInstant Defold extension
     		Module._fbinstant_inited = true;
+            // Set up a progress listener and feed progress to FBInstant
+            ProgressUpdater.addListener(function (percentage) {
```

Comment:
Use new ProgressUpdate from reworked dmloader.js. Also moved into initializeAsync callback to avoid `FBInstant not defined` error.